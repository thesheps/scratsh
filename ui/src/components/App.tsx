import React from "react";
import { Box, Heading, Grommet } from "grommet";

const theme = {
  global: {
    font: {
      family: "Roboto",
      size: "18px",
      height: "20px",
    },
  },
};

export const App = (props) => (
  <Grommet theme={theme}>
    <Box
      tag="header"
      direction="row"
      align="center"
      justify="between"
      background="light-2"
      pad={{ vertical: "small", horizontal: "medium" }}
      elevation="medium"
      {...props}
    >
      <Box direction="row">
        <Heading margin="none">scratSH</Heading>
      </Box>
    </Box>
  </Grommet>
);
