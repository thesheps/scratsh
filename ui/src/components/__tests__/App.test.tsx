import React from "react"
import { render } from '@testing-library/react'
import { App } from "../App"

describe('App', () => {
    it('Matches snapshot', () => {
        const { container } = render(<App />);
        expect(container).toMatchSnapshot();
    });
});