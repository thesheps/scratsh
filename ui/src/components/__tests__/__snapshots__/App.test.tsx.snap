// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Matches snapshot 1`] = `
<div>
  <div
    class="StyledGrommet-sc-19lkkz7-0 kPhklg"
  >
    <header
      class="StyledBox-sc-13pk1d4-0 lkqXsm"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 bIdaiR"
      >
        <h1
          class="StyledHeading-sc-1rdh4aw-0 bGlFVF"
        >
          scratSH
        </h1>
      </div>
    </header>
  </div>
</div>
`;
